File: CSharp/String/Split.cs
using System;

public class Test
{
	public static void Main()
	{
		var str = "伊勢,日向,武蔵";

        var array = str.Split(',');
        
        foreach (var a in array) Console.WriteLine(a); // 伊勢 日向 武蔵


	}
}

File: Java/String/Split.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Test
{
        public static void main (String[] args)
        {       
        	// 変数宣言
        	String str ="沖田,武蔵,モードレッド";
        	    
        	String[] array = str.split(",");
        	    
        	// コンソール出力
                System.out.println(array[0]); // 沖田
                System.out.println(array[1]); // 武蔵
                System.out.println(array[2]); // モードレッド
        }
}
